cfme/tests/cloud/test_providers.py

```diff
@@ -36,6 +41,84 @@ def enable_regions(provider):
     enable_provider_regions(provider)
 
 
+@pytest.fixture(scope='module')
+def az_pwsh_vm():
+    # azure_pwsh contains powershell and necessary scripts
+    provider = get_crud_by_name('rhv42')
```

Comment:
no hardcoding a provider name here, needs to be picked from cfme_data.management_systems via looking for 'powershell_vm'.